server/src/services/trash.service.ts

```diff
@@ -32,7 +40,7 @@ export class TrashService {
     );
 
     for await (const assets of assetPagination) {
-      const ids = assets.map((a) => a.id);
+      const ids = assets.filter((a) => a.trashReason === AssetTrashReason.DELETED).map((a) => a.id);
```

Comment:
```suggestion
      const ids = assets.filter(({ trashReason }) => trashReason === AssetTrashReason.DELETED).map((a) => a.id);
```

Reply:
Thanks. Do you know of a way to do the filter already in the db call? I couldn't find a good way

Reply:
cc @mertalev

Reply:
Should just be able to add `trashReason: AssetTrashReason.DELETED,` below line 38, right?

Reply:
Done!